### PR TITLE
config: add typeorm test to tidb merged ci

### PR DESCRIPTION
**Pull request by Mini256**
Wait PR https://github.com/PingCAP-QE/ci/pull/2492 and https://github.com/PingCAP-QE/tidb-test/pull/2220 merged.

### CODE REVIEW

File: pipelines/pingcap/tidb/latest/merged_integration_typeorm_test.groovy [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                                                bash ${WORKSPACE}/scripts/PingCAP-QE/tidb-test/start_tikv.sh
+                                                export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                                export TIKV_PATH="127.0.0.1:2379"
+                                                export TIDB_TEST_STORE_NAME="tikv"
+                                                cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                            else
+                                                export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                                export TIDB_TEST_STORE_NAME="unistore"
+                                                cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                            fi
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/latest/merged_integration_typeorm_test.groovy:141]: It's better push the script in `tidb-test` repo.

File: pipelines/pingcap/tidb/latest/pod-merged_integration_typeorm_test.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        limits:
+          memory: 16Gi
+          cpu: "6"
+    - name: nodejs
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.16_nodejs-16.x:latest"
+      tty: true
+      resources:
+        requests:
+          memory: 8Gi
+          cpu: "4"
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/latest/pod-merged_integration_typeorm_test.yaml:23]: if requests == limits, then the `requests` filed can be deleted.

File: prow-jobs/pingcap-tidb-latest-postsubmits.yaml [hunk 3]
```diff
@@ -116,6 +116,14 @@ postsubmits:
...
       branches:
         - ^master$
+    - name: pingcap/tidb/merged_integration_typeorm_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: wip/integration-typeorm-test
+      max_concurrency: 1
+      skip_report: true
+      branches:
+        - ^master$
```
Comment (wuhuizuo) [inline prow-jobs/pingcap-tidb-latest-postsubmits.yaml:126]: It is better be a pre-submit type, post submit jobs will just keep build and UT job(for base test coverage uploading)

File: pipelines/pingcap/tidb/latest/merged_integration_typeorm_test.groovy (hunk 1 above)
Comment (Mini256) [reply pipelines/pingcap/tidb/latest/merged_integration_typeorm_test.groovy:141]: I'm not sure why previous tests were written like this

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**